samms_main/audio_mic_helpers.cpp

```diff
@@ -37,6 +37,20 @@ err_t init_audio_struct_and_record (audioClassStruct_t * _audioStruct) {
   set_audio_channel(_audioStruct, AS_CHANNEL_STEREO);
```

Comment:
remove second input from set_audio_channel()

Reply:
since we are passing _audioStruct->.channel within the function set_audio_channel itself